extensions/test/algorithms/Jamfile

```diff
@@ -14,6 +14,7 @@ test-suite boost-geometry-extensions-algorithms
     [ run distance_info.cpp ]
     [ run connect.cpp ]
 #    [ run offset.cpp ]
+    [ run parse.cpp ]
```

Comment:
This is no longer necessary since `parse.cpp` was added in https://github.com/boostorg/geometry/pull/802/